test/Templates.Test/Helpers/AspNetProcess.cs

```diff
@@ -48,9 +61,12 @@ public AspNetProcess(ITestOutputHelper output, string workingDirectory, string p
                     .WaitForExit(assertSuccess: true);
             }
 
+            _httpPort = 5000 + rnd.Next(2000);
+            _httpsPort = 44300 + rnd.Next(1000);
             var envVars = new Dictionary<string, string>
             {
-                { "ASPNETCORE_URLS", "http://127.0.0.1:0" }
+                { "ASPNETCORE_URLS", $"http://localhost:{_httpPort};https://localhost:{_httpsPort}" },
+                { "ASPNETCORE_HTTPS_PORT", $"{_httpsPort}" }
```

Comment:
The plan is for this to go away with aspnet/BasicMiddleware#275

Reply:
Can you also log a bug to track that this line needs to be removed from the test at that time?

Reply:
Yep will do

Reply:
Just checking this is done (searched, but couldn't find any bug covering removing this from the test yet).

Reply:
I was planning to file the bugs at merging time. I'll file them right away

Reply:
> It looks like the HTTPS middleware bits are reintroduced with this PR. I just want to check that this is intentional and this new HTTPS functionality is definitely all ready to ship. Otherwise can we have that as a separate PR?

Yep, I spoke to @DamianEdwards about it yesterday and we've gone through the pertinent reviews and we are all happy so far with it. @danroth27 can confirm

Reply:
Filed https://github.com/aspnet/templating/issues/181